### PR TITLE
Image api research

### PR DESCRIPTION
**Pull request by TheDoplarEffect**
Found an API and reconstructed the original Database with array of objects that also have images.

A lot of the items (all non food) were not detectected.  But a solid 60% now how similar images.
This can be updated later on.

### CODE REVIEW

**Comment by TheDoplarEffect** [comment]
Just waiting for new version of schema to be approved.

**Comment by TheDoplarEffect** [comment]
This PR gives more info on the original Database (Images).
Fixes the seed file to account for new Database.
Also fixees the addItem endpoint since that had already been merged into main.

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import Head from "next/head";
+import Image from "next/image";
```
Comment (NoHara42) [inline src/pages/index.tsx:2]: unused import

File: src/pages/api/addItem.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { NextApiRequest, NextApiResponse } from "next";
 import { z, ZodError } from "zod";
-import { Items, PrismaClient } from "@prisma/client";
+import { MasterItem, PrismaClient } from "@prisma/client";
 
 const prisma = new PrismaClient();
```
Comment (NoHara42) [inline src/pages/api/addItem.ts:5]: see https://github.com/devhausleipzig/camp8-final-project/pull/100#discussion_r1123218794

File: src/pages/api/addItem.ts [hunk 3]
```diff
@@ -13,86 +13,54 @@ export default async function handler(
   if (request.method === "POST") {
     try {
```
Comment (NoHara42) [inline src/pages/api/addItem.ts:14]: please try handling all the errors without nested try/catch blocks, handle all the errors in the parent catch block

**Review by NoHara42 — CHANGES_REQUESTED** [review]
please merge Niels' changes onto this branch
https://github.com/devhausleipzig/camp8-final-project/pull/100

and see suggestions

**Comment by NoHara42** [comment]
love the commit messages by the way

**Review by NoHara42 — APPROVED** [review]
LGTM

**Pull request merged**